docs/signing_release_candidate.md

```diff
@@ -0,0 +1,70 @@
+`verify-install.sh`, that automates the verification process. This script uses a
+set of trusted developer keys (located in the repo under `scripts/keys/`) and
+downloads the necessary data from the GitHub server to verify the integrity of
+the local LND/LNCLI binaries.
+
+This process helps ensure that the release binaries are legitimate and secure.
+
+## Adding a new developer as a signer
+
+When another developer is decided to join the trusted group of people which are
+allowed to sign the LND/LNCLI releases his public PGP key needs to be added to
+the LND repo. This is done via a PR where 2 reviewers make sure the developer
+is in possession of the PGP key which is going to be added to the LND repo.
+(See https://github.com/lightningnetwork/lnd/pull/8788 as an example).
+It is important that the name of the PGP key equals the name in the
+`verify-install` script. See also `scripts/keys/README.md` for more information.
+
+## Signing the new release
+
+Assuming the new developer has been successfully added to the LND repository
+through the aforementioned PR example, they are now able to provide their
+signature for the new release's "Manifest" file. To do so, the developer must
+follow these steps:
+
+* Build the LND/LNCLI software for all platforms/architectures locally running:
+`make docker-release tag=v0.18.3-beta` (`tag=` specifies the version to build,
+here its v0.18.3.beta). For mac users the docker build command might
+fail see https://github.com/lightningnetwork/lnd/issues/9093 for a potential
+solution.
+
+* After a successful build, all binaries, including the Manifest, will be placed
+in a new folder created in the directory where you executed the command. For 
+instance, in the case mentioned above, the folder will be named 
+`lnd-v0.18.3-beta`.
+Ensure that the SHA-256 hashes in your locally generated Manifest match those in
+the Manifest file of the official release on the LND GitHub repository. Only if 
+the hashes are identical should you proceed to sign the release using the 
+following command, assuming your PGP signing key is available on your local 
+device:
+`gpg --local-user KEYID  --detach-sig --output manifest-USERNAME-v0.18.3-beta.sig manifest-v0.18.3-beta.txt`
+
+KEYID being the PGP key you added to the LND repo and USERNAME being the name in
+the `verify-install.sh` script and the name of your PGP key file. Be sure to
+substitute the TAG value `v0.18.3-beta` with the version you are currently 
+signing.
+
+* The last step is to upload the signature file
+(manifest-USERNAME-v0.18.3-beta.sig in the example) to the GitHub release page.
+You will need write permissions to modify the LND release. To avoid interfering
+with other signers who may be updating the GitHub release page, LND developers 
+use a `KeyBase` communication channel to signal when an edit is in progress. 
+Once your signature file is successfully uploaded, the signing process is 
+complete.
```

Comment:
```suggestion
* Finally, upload the signature file
(e.g. manifest-USERNAME-v0.18.3-beta.sig) to the GitHub release page.
Github write permissions are required to upload signatures to the LND release page.
To avoid interfering with other signers who may be updating the GitHub release page, LND developers use a `KeyBase` communication channel to signal when an edit is in progress. 
Once your signature file is successfully uploaded and the release page is unlocked, the
signing process is complete.
```